fallback/default_branch.go

```diff
@@ -0,0 +1,17 @@
+package fallback
```

Comment:
I would probably remove this package and handle at the Yaml level, the same way you are handling the target path. I think it is more flexible